source/_static/scss/includes/_nav.scss

```diff
@@ -479,6 +489,10 @@
 		background-image: linear-gradient(270deg, var(--content-nav-sub-bg) 0%, var(--content-nav-sub-bg) 10%, transparent 100%);
 	}
 
+	.container {
+		padding: 0;
+	}
+
```

Comment:
We need some kind of scroll bar or button to access other K8s platforms on a mobile device.
Right now if you're on Upstream on a mobile device, only Upstream, Openshift, and EKS are accessible platforms. There is no way to move the nav bar to scroll to more options to the right.

Reply:
![image](https://user-images.githubusercontent.com/53910321/217632079-e03a7f5b-853f-46fe-9f7c-4321fe9a9034.png)

Reply:
@djwfyi This is the original design and as per it, we'll have to use the horizontal mouse-wheel incase if we encounter this behaviour on desktop when the window is resized to < 991px. This experience will be mostly on mobile devices, and we can use the native gestures to scroll horizontally.

We can also add conditional arrows on both end at least on the desktop. Let us know your thought as well @oscarocastellanos

Reply:
@rushenn @djwfyi I don't think in mobile, the use of gestures comes as restrictive. Gestures are very intuitive for the majority of mobile users and are widely used on a lot of websites and platforms. Nonetheless, we might want to reduce or avoid any possible conflict or friction that any user might have. Adding arrows for navigation should solve this. Here's a mockup of what I think might work:

![Captura de Pantalla 2023-02-09 a la(s) 10 00 00](https://user-images.githubusercontent.com/89046849/217866985-31cb59b6-b771-4b7c-93f5-777fb3fdbc3c.png)

Thanks for the heads up, Daryl!

Reply:
I like the arrows a lot. Thanks for adding them.